diff -Nru gpsbabel-1.4.4.orig/gui/map.cpp gpsbabel-1.4.4/gui/map.cpp
--- gpsbabel-1.4.4.orig/gui/map.cpp	2012-03-04 18:45:04.000000000 -0400
+++ gpsbabel-1.4.4/gui/map.cpp	2012-10-03 21:50:41.012000262 -0400
@@ -34,6 +34,10 @@
 #include "appname.h"
 #include "dpencode.h"
 
+#ifndef PKGDATADIR
+#define PKGDATADIR "/usr/share/gpsbabel"
+#endif
+
 //------------------------------------------------------------------------
 static QString stripDoubleQuotes(const QString s) {
   QString out;
@@ -60,7 +64,8 @@
   connect(this,SIGNAL(loadFinished(bool)),
 	  this,SLOT(loadFinishedX(bool)));
   this->logTimeX("Start map constuctor");
-  QString baseFile =  QApplication::applicationDirPath() + "/gmapbase.html";
+  QString baseFile =  PKGDATADIR;
+  baseFile += "/gmapbase.html";
   if (!QFile(baseFile).exists()) {
     QMessageBox::critical(0, appName,
 			  tr("Missing \"gmapbase.html\" file.  Check installation"));
