diff -Nru gpsbabel-1.4.4.orig/gui/babeldata.h gpsbabel-1.4.4/gui/babeldata.h
--- gpsbabel-1.4.4.orig/gui/babeldata.h	2010-06-19 19:59:06.000000000 -0400
+++ gpsbabel-1.4.4/gui/babeldata.h	2012-10-03 21:52:01.941000265 -0400
@@ -63,8 +63,8 @@
     upgradeErrors(0),
     upgradeOffers(0),
     runCount(0),
-    startupVersionCheck(true),
-    reportStatistics(true),
+    startupVersionCheck(false),
+    reportStatistics(false),
     allowBetaUpgrades(false),
     ignoreVersionMismatch(false),
     disableDonateDialog(false),
