diff -Nru uuid-1.6.2.orig/Makefile.in uuid-1.6.2/Makefile.in
--- uuid-1.6.2.orig/Makefile.in	2008-03-08 14:31:40.000000000 -0400
+++ uuid-1.6.2/Makefile.in	2013-07-08 23:53:18.920083364 -0400
@@ -62,13 +62,13 @@
 PHP         = @PHP@
 PG_CONFIG   = @PG_CONFIG@
 
-LIB_NAME    = libuuid.la
+LIB_NAME    = libossp-uuid.la
 LIB_OBJS    = uuid.lo uuid_md5.lo uuid_sha1.lo uuid_prng.lo uuid_mac.lo uuid_time.lo uuid_ui64.lo uuid_ui128.lo uuid_str.lo
 
-DCE_NAME    = libuuid_dce.la
+DCE_NAME    = libossp-uuid_dce.la
 DCE_OBJS    = uuid_dce.lo $(LIB_OBJS)
 
-CXX_NAME    = libuuid++.la
+CXX_NAME    = libossp-uuid++.la
 CXX_OBJS    = uuid++.lo $(LIB_OBJS)
 
 PRG_NAME    = uuid
@@ -112,15 +112,15 @@
 	@$(LIBTOOL) --mode=compile $(CXX) $(CPPFLAGS) $(CXXFLAGS) -c $<
 
 $(LIB_NAME): $(LIB_OBJS)
-	@$(LIBTOOL) --mode=link $(CC) -o $(LIB_NAME) $(LIB_OBJS) -rpath $(libdir) \
+	@$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $(LIB_NAME) $(LIB_OBJS) -rpath $(libdir) \
 	    -version-info `$(SHTOOL) version -l c -d libtool $(S)/uuid_vers.h`
 
 $(DCE_NAME): $(DCE_OBJS)
-	@$(LIBTOOL) --mode=link $(CC) -o $(DCE_NAME) $(DCE_OBJS) -rpath $(libdir) \
+	@$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $(DCE_NAME) $(DCE_OBJS) -rpath $(libdir) \
 	    -version-info `$(SHTOOL) version -l c -d libtool $(S)/uuid_vers.h`
 
 $(CXX_NAME): $(CXX_OBJS)
-	@$(LIBTOOL) --mode=link $(CXX) -o $(CXX_NAME) $(CXX_OBJS) -rpath $(libdir) \
+	@$(LIBTOOL) --mode=link $(CXX) $(LDFLAGS) -o $(CXX_NAME) $(CXX_OBJS) -rpath $(libdir) \
 	    -version-info `$(SHTOOL) version -l c -d libtool $(S)/uuid_vers.h`
 
 $(PRG_NAME): $(PRG_OBJS) $(LIB_NAME)
@@ -253,7 +253,7 @@
 	-@if [ ".$(WITH_CXX)" = .yes ]; then \
 	    $(LIBTOOL) --mode=install $(SHTOOL) install -c -m 644 $(CXX_NAME) $(DESTDIR)$(libdir)/; \
 	fi
-	@$(LIBTOOL) --mode=install $(SHTOOL) install -c -s -m 755 uuid $(DESTDIR)$(bindir)/
+	@$(LIBTOOL) --mode=install $(SHTOOL) install -c -m 755 uuid $(DESTDIR)$(bindir)/
 	$(SHTOOL) install -c -m 644 $(S)/uuid.1 $(DESTDIR)$(mandir)/man1/
 	-@if [ ".$(WITH_PERL)" = .yes ]; then \
 	    (cd $(S)/perl && $(MAKE) $(MFLAGS) install DESTDIR=$(DESTDIR)); \
diff -Nru uuid-1.6.2.orig/perl/Makefile.PL uuid-1.6.2/perl/Makefile.PL
--- uuid-1.6.2.orig/perl/Makefile.PL	2007-01-01 14:13:18.000000000 -0400
+++ uuid-1.6.2/perl/Makefile.PL	2013-07-09 00:52:44.243092677 -0400
@@ -33,9 +33,9 @@
 use ExtUtils::MakeMaker;
 
 #   determine source directory
-my ($srcdir) = map { my $d = $_; $d =~ s/\/libuuid\.la$//; $d }
-               grep { -f $_ } ("../libuuid.la", glob("../*/libuuid.la"))
-    or die "no source directory found (where libuuid.la is located)";
+my ($srcdir) = map { my $d = $_; $d =~ s/\/libossp-uuid\.la$//; $d }
+               grep { -f $_ } ("../libossp-uuid.la", glob("../*/libossp-uuid.la"))
+    or die "no source directory found (where libossp-uuid.la is located)";
 
 #   determine extra build options
 my $compat = 0;
@@ -47,7 +47,7 @@
     VERSION_FROM      => 'uuid.pm',
     ABSTRACT_FROM     => 'uuid.pod',
     PREREQ_PM         => {},
-    LIBS              => [ "-L$srcdir/.libs -L$srcdir -luuid" ],
+    LIBS              => [ "-L$srcdir/.libs -L$srcdir -lossp-uuid" ],
     DEFINE            => '',
     INC               => "-I. -I$srcdir",
     PM                => { 'uuid.pm'   => '$(INST_LIBDIR)/uuid.pm',
diff -Nru uuid-1.6.2.orig/php/Makefile.local uuid-1.6.2/php/Makefile.local
--- uuid-1.6.2.orig/php/Makefile.local	2007-01-01 14:13:30.000000000 -0400
+++ uuid-1.6.2/php/Makefile.local	2013-07-09 00:29:38.015089055 -0400
@@ -42,7 +42,7 @@
 
 test: build
 	@version=`$(PHP)-config --version | sed -e 's;^\([0-9]\).*$$;\1;'`; \
-	$(PHP) -q -d "safe_mode=0" -d "extension_dir=./" uuid.ts $$version
+	$(PHP) -q -d "safe_mode=0" -d "extension_dir=./modules/" uuid.ts $$version
 
 install: build
 	@version=`$(PHP)-config --version | sed -e 's;^\([0-9]\).*$$;\1;'`; extdir="$(EXTDIR)"; \
diff -Nru uuid-1.6.2.orig/php/config.m4 uuid-1.6.2/php/config.m4
--- uuid-1.6.2.orig/php/config.m4	2007-05-19 15:26:56.000000000 -0400
+++ uuid-1.6.2/php/config.m4	2013-07-09 00:25:27.507088401 -0400
@@ -33,10 +33,10 @@
 if test "$PHP_UUID" != "no"; then
     PHP_NEW_EXTENSION(uuid, uuid.c, $ext_shared)
     AC_DEFINE(HAVE_UUID, 1, [Have OSSP uuid library])
-    PHP_ADD_LIBPATH([..], )
-    PHP_ADD_LIBRARY([uuid],, UUID_SHARED_LIBADD)
-    PHP_ADD_INCLUDE([..])
+    PHP_ADD_LIBRARY([ossp-uuid],, UUID_SHARED_LIBADD)
+    PHP_ADD_INCLUDE([../uuid-1.6.2])
     PHP_SUBST(UUID_SHARED_LIBADD)
+    LDFLAGS="$LDFLAGS -L../uuid-1.6.2/.libs"
 
     dnl  avoid linking conflict with a potentially existing uuid_create(3) in libc
     AC_CHECK_FUNC(uuid_create,[
diff -Nru uuid-1.6.2.orig/php/uuid.c uuid-1.6.2/php/uuid.c
--- uuid-1.6.2.orig/php/uuid.c	2007-01-01 14:35:57.000000000 -0400
+++ uuid-1.6.2/php/uuid.c	2013-07-09 00:27:27.492088714 -0400
@@ -496,7 +496,7 @@
 }
 
 /* module function table */
-static function_entry uuid_functions[] = {
+static zend_function_entry uuid_functions[] = {
     PHP_FE(uuid_create,  NULL)
     PHP_FE(uuid_destroy, NULL)
     PHP_FE(uuid_clone,   NULL)
diff -Nru uuid-1.6.2.orig/php/uuid.ts uuid-1.6.2/php/uuid.ts
--- uuid-1.6.2.orig/php/uuid.ts	2007-01-01 14:13:33.000000000 -0400
+++ uuid-1.6.2/php/uuid.ts	2013-07-09 00:30:09.971089095 -0400
@@ -36,7 +36,7 @@
 
 print "++ loading DSO uuid.so (low-level API)\n";
 if (!extension_loaded('uuid')) {
-    dl('modules/uuid.so');
+    dl('uuid.so');
 }
 
 print "++ loading PHP uuid.php${php_version} (high-level API)\n";
diff -Nru uuid-1.6.2.orig/uuid-config.in uuid-1.6.2/uuid-config.in
--- uuid-1.6.2.orig/uuid-config.in	2008-03-07 06:49:59.000000000 -0400
+++ uuid-1.6.2/uuid-config.in	2013-07-08 23:57:07.572083978 -0400
@@ -121,7 +121,7 @@
             output_extra="$output_extra $uuid_ldflags"
             ;;
         --libs)
-            output="$output -luuid"
+            output="$output -lossp-uuid"
             output_extra="$output_extra $uuid_libs"
             ;;
         * )
diff -Nru uuid-1.6.2.orig/uuid.pc.in uuid-1.6.2/uuid.pc.in
--- uuid-1.6.2.orig/uuid.pc.in	2008-03-07 06:49:59.000000000 -0400
+++ uuid-1.6.2/uuid.pc.in	2013-07-08 23:57:40.465084049 -0400
@@ -36,7 +36,7 @@
 Description: Universally Unique Identifier (UUID) Library
 Version: @UUID_VERSION_RAW@
 URL: http://www.ossp.org/pkg/lib/uuid/
-Cflags: -I${includedir}
-Libs: -L${libdir} -luuid
+Cflags: -I${includedir}/ossp
+Libs: -L${libdir} -lossp-uuid
 Libs.private: @LIBS@
 
